frontend/public/components/nav.jsx

```diff
@@ -312,16 +325,31 @@ export const Navigation = ({ isNavOpen, onNavSelect }) => {
 
         <NavSection title="Catalog">
           <HrefLink href="/catalog" name="Developer Catalog" activePath="/catalog/" />
-          <ResourceNSLink model={ClusterServiceVersionModel} resource={ClusterServiceVersionModel.plural} name="Installed Operators" />
           <HrefLink
             href="/provisionedservices"
             name="Provisioned Services"
             activePath="/provisionedservices/"
             startsWith={provisionedServicesStartsWith}
             required={FLAGS.SERVICE_CATALOG}
           />
-          <HrefLink required={FLAGS.OPERATOR_HUB} href="/operatorhub" name="OperatorHub" activePath="/operatorhub/" />
+          <ResourceNSLink
+            model={ClusterServiceVersionModel}
+            resource={ClusterServiceVersionModel.plural}
+            name="Installed Operators"
+            separatorConditions={[
```

Comment:
Annoying that we have to include this logic just because of the way Patternfly couples the separator with the `<NavItem>` component.

Reply:
Yeah, we would basically need this login somewhere either way. I did file an issue against PF-Next though.